units.cpp

```diff
@@ -18,6 +18,7 @@ void Unit::dealDamageTo(Unit& damagedUnit){
 		 m_maxHP = std::round(1.1*m_maxHP);
 		 m_hp = m_maxHP;
 		 m_damage = std::round(1.1*m_damage);
+		 m_attackspeed = 1.1*m_attackspeed;
```

Comment:
Ez ránézésre jó, de most szegény harcos a szintlépéstől 10%-kal lassabb lett. Ezért fontos a változók helyes elnevezése. ☝️ Amit attackspeed néven tároltok, az valójában egy cooldown, aminek meg csökkennie kéne szintlépésnél. Nevezd is át, és a getter metódusát is!